examples/cifar10_cnn.py

```diff
@@ -99,3 +109,37 @@
                         steps_per_epoch=x_train.shape[0] // batch_size,
                         epochs=epochs,
                         validation_data=(x_test, y_test))
+
+# Save model and weights
+ensure_dir_exists(save_dir)
+model.save(save_dir+model_name)
+model.save_weights(save_dir+weights_name)
+print('Saved trained model: ' +save_dir+model_name)
+print('Saved weights: ' +save_dir+weights_name)
+
+# Load label names to use in prediction results
+label_list = os.path.expanduser("~") + '/.keras/datasets/cifar-10-batches-py/batches.meta'
```

Comment:
This will break for Windows users. Check how we construct file paths elsewhere (e.g. in `backend/common.py`).